src/osvr/ClientKit/ContextC.cpp

```diff
@@ -69,3 +70,14 @@ OSVR_ReturnCode osvrClientShutdown(OSVR_ClientContext ctx) {
     osvr::common::deleteContext(ctx);
     return OSVR_RETURN_SUCCESS;
 }
+
+OSVR_ReturnCode osvrClientLog(OSVR_ClientContext ctx, OSVR_LogLevel severity,
+                              const char *message) {
+    if (!ctx) {
+        return OSVR_RETURN_FAILURE;
```

Comment:
I think we can do better - if there's no context, then the best we can do is what we'd do before - std::cerr. Let's not make people error handle their logging.